edge/pkg/clusterd/mission_deployer.go

```diff
@@ -43,44 +45,93 @@ func NewMissionDeployer() *MissionDeployer {
 	}
 }
 
+func resolveCommand(input string) string {
+	output := strings.ReplaceAll(input, "[kubectl]", config.Config.KubectlCli)
+	output = strings.ReplaceAll(output, "[kubeconfig]", config.Config.Kubeconfig)
+
+	return output
+}
+
+func deployMisionContent(mission *edgeclustersv1.Mission) {
+	if strings.TrimSpace(mission.Spec.MissionResource) != "" {
+		deployContentCmd := fmt.Sprintf("printf '%s' | %s apply --kubeconfig=%s -f - ", mission.Spec.MissionResource, config.Config.KubectlCli, config.Config.Kubeconfig)
+		output, err := helper.ExecCommandToCluster(deployContentCmd)
+		if err != nil {
+			klog.Errorf("Failed to deploy the resource of mission %v: %v", mission.Name, err)
```

Comment:
should the function just return here when this error happens?

Reply:
if yes, we could remove the “else" on line 61

Reply:
Updated as suggested.